mkosi/config.py

```diff
@@ -2879,8 +2890,10 @@ def load_environment(args: argparse.Namespace) -> dict[str, str]:
     if dnf := os.getenv("MKOSI_DNF"):
         env["MKOSI_DNF"] = dnf
 
-    for s in args.environment:
+    entries = [x for f in args.environment_files for x in f.read_text().strip().splitlines()]
```

Comment:
Nit: `x` is a very generic choice, let's give them proper names, to make this clearer and not much longer
```suggestion
    entries = [line for envfile in args.environment_files for line in f.read_text().strip().splitlines()]
```
(or at least give it a more telling short name like `e` or `l`, but proper names are nicer)